### PR TITLE
Adding info and error popup notifications

### PR DESCRIPTION
**Pull request by maaverik**
Adding info popups for leave and delete queue operations similar to error popups.

### CODE REVIEW

File: simplq/src/components/ErrorHandler.jsx [hunk 1]
```diff
@@ -25,7 +22,7 @@ export class ErrorBoundary extends React.Component {
     // You can also log the error to an error reporting service
     // logErrorToMyService(error, errorInfo);
     console.log('In componentDidCatch');
-    store.dispatch(setErrorNotifOpen(true));
+    store.dispatch(setErrorPopupMessage(true));
```
Comment (daltonfury42) [inline simplq/src/components/ErrorHandler.jsx:25]: Wanted to understand, why do we set this to "true" here?

Comment (daltonfury42) [reply simplq/src/components/ErrorHandler.jsx:25]: We are not using `error` and `errorInfo` here? Or maybe these should not be a popul, we should let them go down and be caught by sentry?

**Review by daltonfury42 — COMMENTED** [review]
This is a nice one.

**Review by daltonfury42 — APPROVED** [review]

File: simplq/src/components/ErrorHandler.jsx (hunk 1 above)
Comment (maaverik) [reply simplq/src/components/ErrorHandler.jsx:25]: What I remember from when I wrote this is that the code never reaches this point. That's probably why I didn't notice this was set to true. I'll try to figure this out soon.

Comment (maaverik) [reply simplq/src/components/ErrorHandler.jsx:25]: Do you want to merge this now and use this also for the prod branch?

**Pull request merged**